src/components/booking-portal/ActiveBookings.vue

```diff
@@ -103,7 +116,10 @@
 </template>
 
 <script>
-import { getPaymentPeriodicityLabel } from '@/constant/enums';
+import {
+    getPaymentPeriodicityLabel,
```

Comment:
why we need getPaymentPeriodicityLabel?

Reply:
This is from before, and we have used it for the periodicity column.